### PR TITLE
Proposal Details 

### LINKED ISSUE #2190: Proposal Details Page `Read Only` - Design Specification
**Describe the task that needs to be done**
Once @hexyls has implemented the creation of a liquidity rewards proposal (https://github.com/protofire/omen-exchange/issues/2044) we should be able to implement the proposal details page which shows proposal info and market infos. This version of the proposals details view is only `read only` without any voting write functions:

proposal-detail-view

Notes:
- Replace transaction modal with confirmed state with a redirect to this `Proposal Detail Page`.
- This version is only a read-only page with proposal data and market data.

Figma:
[Design Specification [Desktop]](https://www.figma.com/file/PhYs1uw1LgqpvriqHDPl7k/Playground-FINAL?node-id=402%3A13195)
[Design Specification [Mobile]](https://www.figma.com/file/PhYs1uw1LgqpvriqHDPl7k/Playground-FINAL?node-id=402%3A12460)

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #2190 
How to test:
Because data is not connected we can only test styles so we need to make sure that designs are same as on figma:
- To access page go to http://localhost:3000/#/proposal-details-page
- Liquidity rewards section(mobile and desktop) https://www.figma.com/file/zVniQrYgkj8AfQs5zNeBrS/Components-FINAL?node-id=300%3A1290
- Whole page(mobile and desktop https://www.figma.com/file/PhYs1uw1LgqpvriqHDPl7k/Playground-FINAL?node-id=402%3A12460
- [ ] Mobile
- [ ] Desktop

### CODE REVIEW

File: app/src/components/common/table/responsive_table_simple/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TYPE } from '../../../../theme'
+
+const Table = styled.div`
+  width: 100%;
+  box-shadow: 0px 2px 4px rgb(0 0 0 / 5%);
```
Comment (pimato) [inline app/src/components/common/table/responsive_table_simple/index.tsx:8]: lets remove the box shadow here

**Review by pimato — CHANGES_REQUESTED** [review]

**Comment by pimato** [comment]
Also remove the box shadhow here:
Bildschirmfoto 2021-08-25 um 12 37 47

and here:
Bildschirmfoto 2021-08-25 um 12 38 01

**Comment by pimato** [comment]
Bildschirmfoto 2021-08-25 um 12 39 24

should be `primary1`

**Comment by pimato** [comment]
the padding at the top in the tooltip should be removed:
Bildschirmfoto 2021-08-25 um 12 41 39

Bildschirmfoto 2021-08-25 um 12 41 37

**Comment by pimato** [comment]
can we add hover and onClick styling for the navigation back:
ArrowButton

Figma:
[Design Specification](https://www.figma.com/file/zVniQrYgkj8AfQs5zNeBrS/Components-FINAL?node-id=17%3A10)

**Comment by pimato** [comment]
can we add hover and onClick styling for the navigation back:
ArrowButton

Figma:
[Design Specification](https://www.figma.com/file/zVniQrYgkj8AfQs5zNeBrS/Components-FINAL?node-id=17%3A10)

**Comment by pimato** [comment]
looks like only the arrow has the hand icon on Hover:
https://user-images.githubusercontent.com/5337809/130804301-258c08bb-3b0f-4d5e-813f-d39f3760800b.mov

**Review by pimato — APPROVED** [review]

**Review by hexyls — APPROVED** [review]
Looks good man!

**Pull request merged**